Example/WordPress-Aztec-iOS/EditorDemoController.swift

```diff
@@ -5,20 +5,20 @@ import Aztec
 
 class EditorDemoController: UIViewController
 {
-    private var bottomConstraint: NSLayoutConstraint?
+    private var bottomConstraint: NSLayoutConstraint!
 
 
     private (set) lazy var editor: AztecVisualEditor = {
-        let e = AztecVisualEditor(textView: self.textView)
+        let e = AztecVisualEditor(textView: self.richTextView)
```

Comment:
I know this wasn't introduced by your PR but I'd take the opportunity to remove the `e` variable, both for having a non standard name, and for not being really necessary.